app/background/node/service.js

```diff
@@ -473,7 +559,26 @@ export class NodeService extends EventEmitter {
       throw new Error('No client.');
   }
 
-  async _execRPC(method, args) {
+  async _execHostedRPC(method, args) {
+    const json = await hapiPost('', {
+      method,
+      params: args,
+    });
+
+    if (!json) {
+      throw new Error('No body for JSON-RPC response.');
+    }
+
+    return json.result;
+  }
+
+  async _execRPC(method, args, proxy) {
+    if (proxy) {
```

Comment:
Why would we ever use the proxy if not in SPV mode?

Reply:
True. I will rename variable

Reply:
mm maybe i misunderstand. im not asking about the variabel name but why you need to check BOTH `proxy` AND `this.getSpvMode()` ?

Reply:
oh! this is actually for the rpc method that we DONT want to proxy to hosted node on SPV, such as `verifymessage`, `sendrawtransactions`, `sendrawairdrop`, etc.